sensor_msgs/PointCloud2 pointcloud
int64 x
int64 y
---
float64 x
float64 y
float64 z